src/components/Graphs/index.jsx

```diff
@@ -48,6 +54,7 @@ const Graphs = ({ classes, benchmarkData, overviewMode }) => (
         <div key={title}>
           <h2 className={classes.benchmarkTitle}>{title}</h2>
           <a href={jointUrl} target="_blank" rel="noopener noreferrer"><LinkIcon className={classes.linkIcon} /></a>
+          <a href={`/${platform}/${title.replace(/\s+/g, '-').toLowerCase()}`} rel="noopener noreferrer"><ArrowDownward className={classes.linkIcon} /></a>
```

Comment:
You can use `benchmark` instead of `title`.

Can you please make this `a` element dependent to if `overviewMode` === 'overview'?